<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>440</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>NotePuppy</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="0" colspan="2">
     <widget class="QWidget" name="buttonWdiget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>60</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>51</height>
       </size>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>800</width>
         <height>53</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="buttonNewOne">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>51</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Lucida Grande</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>NEW</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonImport">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>51</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Lucida Grande</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>IMPORT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonExport">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>51</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Lucida Grande</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>EXPORT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonRename">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>51</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Lucida Grande</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>RENAME</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDelete">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>51</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Lucida Grande</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>DELETE</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="animated">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="headerHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">3</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionNew_Quick_File"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="actionRename"/>
    <addaction name="actionDelete"/>
    <addaction name="actionClear_Selection"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Hide_Buttons"/>
    <addaction name="actionTranslucent"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create a new file or folder.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>&amp;Import</string>
   </property>
   <property name="toolTip">
    <string>Import a file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>&amp;Export</string>
   </property>
   <property name="toolTip">
    <string>Export a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>&amp;Rename</string>
   </property>
   <property name="toolTip">
    <string>Change the name or position</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="toolTip">
    <string>Ditch something</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Backspace</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionNotePuppy_Help">
   <property name="text">
    <string>NotePuppy Help</string>
   </property>
  </action>
  <action name="actionAbout_NotePuppy_2">
   <property name="text">
    <string>About NotePuppy</string>
   </property>
  </action>
  <action name="actionShow_Hide_Buttons">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show &amp;Buttons</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionClear_Selection">
   <property name="text">
    <string>Clear Selection</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionTranslucent">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Translucent</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionNew_Quick_File">
   <property name="text">
    <string>New Quick File</string>
   </property>
   <property name="toolTip">
    <string>Creates a file in the 'Quick' Folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDelete</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExport</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>exportFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionImport</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>importFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRename</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>renameFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_Hide_Buttons</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>showHideButtons(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClear_Selection</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTranslucent</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setTranslucent(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_Quick_File</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>quickFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newFile()</slot>
  <slot>openFile()</slot>
  <slot>importFile()</slot>
  <slot>exportFile()</slot>
  <slot>renameFile()</slot>
  <slot>deleteFile()</slot>
  <slot>showHideButtons(bool)</slot>
  <slot>clearSelection()</slot>
  <slot>setTranslucent(bool)</slot>
  <slot>quickFile()</slot>
 </slots>
</ui>
